config/otel-collector-config.yaml

```diff
@@ -0,0 +1,37 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  prometheus:
+    endpoint: "0.0.0.0:8889"
+
+  logging:
+
+  jaeger:
+    endpoint: jaeger:14250
+    tls:
+      insecure: true
+
+processors:
+  batch:
+
+extensions:
+  health_check:
+  pprof:
+    endpoint: :1888
+  zpages:
+    endpoint: :55679
```

Comment:
I can think of a use case for using health check but do we have a use case for using pprof and zpages?
If not I would remove both extensions in order to keep the overall configuration at a minimum.